openapi/v2/core-metadata.yaml

```diff
@@ -1243,6 +1252,55 @@ paths:
             application/json:
               schema:
                   $ref: '#/components/schemas/ErrorResponse'
+    put:
```

Comment:
`put` is correct for this version, but the othe json Update Request version should be a `patch`. Currently it is a `put`.

Reply:
Modified.